### ISSUE TITLE
Proxies Not Working

### ISSUE THREAD

**Issue by the-saint** [GetiPlayerAutomator/get-iplayer-automator#492]
Provided proxies have not been working for me for about a week. I can get a public proxy to work but only after trying many of them and they don''t work for very long. Sometimes no proxies work at all. I'm not sure what's going on.

**Comment by dinkypumpkin** [comment]
Duplicate of #483
